### DISCUSSION TITLE
Unable to change child element of _Expander_

### DISCUSSION THREAD

**Discussion by chkr1011** [AvaloniaUI/Avalonia#7530] [Q&A] [answered]
I am trying to set a different value for the Padding of an internal control of the _Expander_.

But unfortunately the style is not applied at all. Already tried declaring Style at App.axaml, Window element, Expander element etc.

Style reference: 
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/Expander.xaml

Style:
```
<Style Selector="Expander /template/ ToggleButton#ExpanderHeader /template/ Border#ExpandCollapseChevronBorder">
    <Setter Property="Margin"
            Value="0" />
</Style>
```

Nothing is happening. The style is simply ignored. What am I doing wrong here!?

**Comment by maxkatz6** [comment, marked as answer]
Margin is set with local value priority there, so it can't be changed with styles
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/Expander.xaml#L112

For reasoning of this behavior you can read this article
https://docs.avaloniaui.net/docs/styling/troubleshooting
Specifically "Locally set Properties override Styles" part.

**Reply by maxkatz6** [reply]
Also we plan to relax this restriction for values set inside of templates. But it won't be available before next major release.

**Reply by chkr1011** [reply]
Thanks for explanation. So my only solution is to copy the original style into my project and doing the changes manually right?

**Reply by maxkatz6** [reply]
As for now - yes. But you only need template of this button.
